src/util.ts

```diff
@@ -101,13 +103,23 @@ export function isCacheFeatureAvailable(): boolean {
 }
 
 export function getVersionFromFileContent(
+  fileName: string,
   content: string,
   distributionName: string
 ): string | null {
-  const javaVersionRegExp = /(?<version>(?<=(^|\s|\-))(\d+\S*))(\s|$)/;
-  const fileContent = content.match(javaVersionRegExp)?.groups?.version
-    ? (content.match(javaVersionRegExp)?.groups?.version as string)
-    : '';
+  let fileContent = null;
```

Comment:
I suppose you wanted to keep the amount of changes minimal but I think the name `fileContent` is now slightly misleading, so I suggest something like `parsedVersion` instead.